### PR TITLE
Adding cloud features

### PR DESCRIPTION
**Pull request by Bidek56**
Adding cloud features

### CODE REVIEW

**Comment by mostopalove** [comment]
Hi @Bidek56 !
I hope you're doing well. I was wondering if you could provide an update on when this PR might be merged and published.
Thank you!

**Comment by Bidek56** [comment]
@universalmind303 any ETA? Thx

File: polars/types.ts [hunk 1]
```diff
@@ -141,12 +141,21 @@ export interface ReadParquetOptions {
...
   cache?: boolean;
+  parallel?: "auto" | "columns" | "row_groups" | "none";
+  glob?: boolean;
+  hivePartitioning?: boolean;
+  hiveSchema?: unknown;
+  tryParseHiveDates?: boolean;
   rechunk?: boolean;
+  lowMemory?: boolean;
+  useStatistics?: boolean;
+  cloudOptions?: unknown;
```
Comment (impatient) [inline polars/types.ts:156]: Why the type change here?  

I'd needed it previously and noticed it wasn't passed in to Rust (Got around it with env variables).  Also FWIW, was reading parquet files from GCS and S3 without the polars feature changes.

File: Cargo.toml [hunk 2]
```diff
@@ -155,7 +155,11 @@ features = [
   "string_pad",
   "replace",
   "cov",
-  "http"
+  "http",
```
Comment (impatient) [inline Cargo.toml:158]: Oooh.. you took these out here and now you're adding them back, so inadvertently took away features here.

https://github.com/pola-rs/nodejs-polars/pull/253/files#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542L158-L162

File: polars/types.ts (hunk 1 above)
Comment (Bidek56) [reply polars/types.ts:156]: I change the type, b/c Javascript Map was not getting passed to the Rust code.

**Pull request merged**